src/tests/builders/version.rs

```diff
@@ -56,8 +56,8 @@ impl<'a> VersionBuilder<'a> {
     }
 
     /// Sets the version's `license` value.
-    pub fn license(mut self, license: Option<&'a str>) -> Self {
-        self.license = license;
+    pub fn license(mut self, license: impl Into<String>) -> Self {
+        self.license = Some(license.into());
         self
     }
```

Comment:
and if you're wondering if I don't have to adjust all of the call sites? you're right, but this function isn't actually used anywhere at the moment so the perf hit is basically zero 😂